include/aikido/distance/RandomConfigurationRanker.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef AIKIDO_DISTANCE_RANDOMCONFIGURATIONRANKER_HPP_
+#define AIKIDO_DISTANCE_RANDOMCONFIGURATIONRANKER_HPP_
+
+#include "aikido/distance/ConfigurationRanker.hpp"
+
+namespace aikido {
+namespace distance {
+
+class RandomConfigurationRanker : public ConfigurationRanker
```

Comment:
Is this class required, since the configurations are anyway generated in a random order (except maybe when sequential sampleable with start state is used).

Reply:
I don't think this is necessary, unless we decide that we want to have a "default" ranker (and we don't want that defualt to be something like `NominalConfigurationRanker`).